mlflow/tracking/client.py

```diff
@@ -1422,7 +1424,7 @@ def _is_plotly_figure(fig):
     def log_image(
         self,
         run_id: str,
-        image: Union["numpy.ndarray", "PIL.Image.Image"],
+        image: Union["numpy.ndarray", "PIL.Image.Image", Image],
```

Comment:
```suggestion
        image: Union["numpy.ndarray", "PIL.Image.Image", "Image"],
```
I think this will solve your docstring issue. Sphinx doesn't have the full context of resolving this dependency type.